src/main/scala/com/mozilla/telemetry/pings/package.scala

```diff
@@ -46,7 +46,36 @@ package object pings {
 
   case class NewStyleExperiment(branch: String)
 
-  case class Addons(activeExperiment: Option[OldStyleExperiment])
+  case class ActiveAddon(isSystem: Option[Boolean], isWebExtension: Option[Boolean]) {
+    def isQuantumReady: Boolean = {
+      this.isSystem orElse this.isWebExtension getOrElse false
```

Comment:
I treat this differently. If neither `isSystem` or `isWebExtension` are available, then `isQuantumReady` is `null` since it is unknown.